root/genre/NotFound.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import NotFound from '../components/NotFound';
+
+const GenreNotFound = () => (
+  <NotFound title={l('Genre Not Found')}>
+    <p>
+      {l('Sorry, we could not find a genre with that MusicBrainz ID')}
```

Comment:
Final stop `.` is missing.

Maybe linking to the genre list page and/or tag search would be helpful?

Reply:
Done :)

Reply:
Perfect!